### DISCUSSION TITLE
how to add a book to the database?

### DISCUSSION THREAD

**Discussion by EttaK** [bookwyrm-social/bookwyrm#3252] [Q&A] [answered]
I'm searching for books that have been published and not finding them all in the database. How do I add books? I'm computer stupid so I may not be using the correct terminology -- what I mean is, when I search for a book on the main page search, I'm not finding a book that I know has been published, and want to add it. Thanks

**Comment by mouse-reeve** [comment, marked as answer]
Happily, we have some documentation on that: docs.joinbookwyrm.com/adding-books.html

**Reply by EttaK** [reply]
Thank you!
